pages/download.html

```diff
@@ -11,16 +11,18 @@
 <div style="padding: 16px 32px">
 	<h3>Redirecting...</h3>
 	<p>
-		If you are not being redirected automatically, <a href="/download/windows/" class="set-os-download-url">click here</a>.
+		If you are not being redirected automatically, <a href="/download/windows" class="set-os-download-url">click here</a>.
```

Comment:
you should keep the trailing `/` here
```suggestion
		If you are not being redirected automatically, <a href="/download/windows/" class="set-os-download-url">click here</a>.
```